tox.ini

```diff
@@ -52,8 +52,8 @@ deps =
 	coverage
 commands =
 	python -c "import fcntl; fcntl.fcntl(1, fcntl.F_SETFL, 0)"
-	pip install -r requirements/tests.txt
 	pip install -e .
+	pip install -r requirements/tests.txt
```

Comment:
`pip install -e .` already installs the requirements, so this line should not be needed.

Reply:
Nice catch, will adapt it

Reply:
Sorry, actually it is neeeded as `pip install -e .` also only installs dependencies needed for development, so for test `requirements/tests.txt` needs to be installed too

Reply:
`pip install -e .` only installs packages we declared in the `install_requires` argument inside `setup.py`. So it will only install the packages we specified in `base.txt`, in our case.

Reply:
Understood, thanks!